<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sendfile</class>
 <widget class="QDialog" name="Sendfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(85, 170, 255);
}
QTableWidget{
	color: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QLabel" name="label_Recv">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 9pt &quot;等线&quot;;
background-color: rgb(115, 115, 115);
border-top:0px solid #e8f3f9;
background:  transparent;</string>
   </property>
   <property name="text">
    <string>接收进度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Send">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 9pt &quot;等线&quot;;
background-color: rgb(115, 115, 115);
border-top:0px solid #e8f3f9;
background:  transparent;</string>
   </property>
   <property name="text">
    <string>发送进度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Open">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 9pt &quot;等线&quot;;
	color: rgb(0, 0, 0);
 border-style: outset;
     border-width: 1px;
     border-radius: 6px;
     border-color: beige;    
     padding: 6px;
	
	background-color: rgb(137, 194, 255);
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
                   }
QPushButton:hover{
	
	
	
	background-color: rgb(162, 212, 255);
                   border-color:rgba(255,255,255,200);
                   
	color: rgb(0, 0, 0);
                   }</string>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Send">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 9pt &quot;等线&quot;;
	color: rgb(0, 0, 0);
 border-style: outset;
     border-width: 1px;
     border-radius: 6px;
     border-color: beige;    
     padding: 6px;
	
	background-color: rgb(137, 194, 255);
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
                   }
QPushButton:hover{
	
	
	
	background-color: rgb(162, 212, 255);
                   border-color:rgba(255,255,255,200);
                   
	color: rgb(0, 0, 0);
                   }</string>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_send">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_receive">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>421</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/发送文件.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>tableWidget</zorder>
  <zorder>label_Recv</zorder>
  <zorder>label_Send</zorder>
  <zorder>pushButton_Open</zorder>
  <zorder>pushButton_Send</zorder>
  <zorder>progressBar_send</zorder>
  <zorder>progressBar_receive</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
